client_cli/pytests/src/client_cli/configuration.py

```diff
@@ -63,20 +66,25 @@ def generate_by_peers(self, peers_configs_dir):
 
         for port in range(self.port_min, self.port_max + 1):
             config_copy = self._config.copy()
-            config_copy["TORII_API_URL"] = f"http://localhost:{port}"
-            file_name = f"config_to_peer_{port}.json"
+            parsed_url = urlparse(config_copy["torii_url"])
+            updated_url = parsed_url._replace(
+                netloc=f"{parsed_url.hostname}:{port}"
+            ).geturl()
+            config_copy["torii_url"] = updated_url
+
+            file_name = f"config_to_peer_{port}.toml"
```

Comment:
Why this all is needed?

You can use the same client configuration file each time and override `TORII_API_URL` for each execution separately.

Reply:
Agree. Removed it.